cli.js

```diff
@@ -2062,11 +2064,12 @@ program
         const secret = options.secret;
         const channels = !options.channels || options.channels.split(',').map(item => item.trim());
         const scopes = !options.scopes || options.scopes.split(',').map(item => item.trim());
-        const redirecturis = !options.redirecturis || options.redirecturis.split(',').map(item => item.trim());
+        const redirecturis = options.redirecturis ? options.redirecturis.split(',').map(item => item.trim()) : [];
```

Comment:
needs to be set to an empty array if the client does not have redirectURLs (passwordless login)